sockeye/arguments.py

```diff
@@ -821,10 +838,25 @@ def add_training_args(params):
                               help='How to normalize the loss. By default loss is normalized by the number '
                                    'of valid (non-PAD) tokens (%s).' % C.LOSS_NORM_VALID)
 
+    train_params.add_argument('--length-task',
+                              type=str,
+                              default=None,
+                              choices=[C.LENGTH_TASK_RATIO, C.LENGTH_TASK_LENGTH],
+                              help='If not None, adds an auxiliary task during training to predict source/target length ratios '
```

Comment:
As the help string is regarding the CLI I think it would be better to write `if specified ...` instead of `If not None` as there's no `None` on the CLI.

Reply:
Agree, will fix in the next commit.